### ISSUE TITLE
Anti-GluD2

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10543]
**Preferred term label:**
Autoimmune antibody positivity HP:0030057

**Synonyms**
Glutamate receptor δ2 serum antibodies
Anti-Glutamate Receptor δ2 Antibodie
glutamate receptor delta2 autoantibody 

**Definition (free text, please give PubMed ID)**
The presence of autoantibodies in the serum that react against glutamate receptor delta2.

PMID: 17049194 Glutamate receptor delta2, a member of the glutamate receptor family, is predominantly expressed in cerebellar Purkinje cells and plays a crucial role in cerebellar functions. To date **anti-GluRdelta2 autoantibody** was detected in a patient with chronic cerebellitis. Herein, an 18-month-old boy presented with cerebellar ataxia 9 days following a mild respiratory tract infection. Although cerebellar ataxia gradually improved, it worsened yet again following mumps and varicella virus infection. Cerebro-spinal fluid examination and magnetic resonance imaging of the brain demonstrated pleocytosis and meningeal enhancement, respectively. Furthermore, **glutamate receptor delta2 autoantibody** was detected in serum and cerebro-spinal fluid. Thus, we believe that the glutamate receptor delta2 autoantibody may play a role in cerebellar ataxia and consecutive cerebellitis. 

PMID: 36696031 We report two novel cases of autoimmune cerebellar ataxia (ACA) associated with a**nti-glutamate receptor δ2 antibodies** (Gluδ2-Abs)(....) The present cases suggested that ACA associated with Gluδ2-Abs is an extremely rare but treatable disease. Therefore, testing for Gluδ2-Abs might be considered in the setting of suspected ACA and no initial antibody identification. 

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:624259 Non-specific autoimmune cerebellar ataxia with characteristic antibodies

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6000876
- label: Anti-Glutamate Receptor delta2 antibody positivity
- parents: [Autoimmune antibody positivity (HP:0030057)](https://hpo.jax.org/app/browse/term/HP:0030057)
- pmid: PMID:17049194|PMID:36696031
- def: The presence of autoantibodies in the serum that react against glutamate receptor delta2.
- comment: Glutamate receptor delta2, a member of the glutamate receptor family, is predominantly expressed in cerebellar Purkinje cells and plays a crucial role in cerebellar functions. Anti-GluRdelta2 autoantibody has been detected in a patient with chronic cerebellitis.
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10543
- synonyms:
